base/dsfs/dataset_test.go

```diff
@@ -999,3 +1047,41 @@ func BenchmarkValidateJSON(b *testing.B) {
 		})
 	}
 }
+
+// func BenchmarkPrepareDataset1000000Rows(b *testing.B) {
+// 	ctx := context.Background()
+// 	_, ds := GenerateDataset(b, 1000000, "csv")
+// 	fs := qfs.NewMemFS()
```

Comment:
I guess this will be uncommented before the merge?

Reply:
actually, needs to be deleted. `PrepareDataset` no longer exists